ui/pl.json

```diff
@@ -0,0 +1,24 @@
+{
+  "building-delete": "Usuń",
+  "building-move": "Przesuń",
+  "contact_completed": "Kontrakt został wypełniony.",
+  "contact_unlocked": "Nowy kontrakt został odblokowany.",
+  "equipment_unknown": "Błąd: Nieznany ekwipunek.", //Review and context needed here!
+  "equipment_type_unknown": "Błąd: Nieznany typ ekwipunku.", //Review and context needed here!
+  "equipment_space_already_occupied": "Błąd: To miejsce zostało już zajęte.", //Review and context needed here!
```

Comment:
This is an error sent from the server when you try to move/build in the space where something else is already existing.